spec/macros/resource_macros.rb

```diff
@@ -50,40 +50,38 @@ def it_should_be_updatable(attribute, value = "TESTDATA", extra = {})
     context "update", :vcr do
       before(:all) do
         VCR.use_cassette("#{described_class.to_s}_update_create") do
-          @object = described_class.create(client, valid_attributes.merge(default_options))
+          @updatable_object = described_class.create!(client, valid_attributes.merge(default_options))
         end
       end
 
       before(:each) do
-        @object.send("#{attribute}=", value)
-        extra.each { |k, v| @object.send("#{k}=", v) }
+        @updatable_object.send("#{attribute}=", value)
+        extra.each { |k, v| @updatable_object.send("#{k}=", v) }
       end
 
       it "should be savable" do
-        expect(@object.save).to be(true)
+        expect(@updatable_object.save!).to be(true)
```

Comment:
In this case I think it should be save without the bang.

You could try:

```suggestion
    expect(object.save).to eq(true), "Expected object to save, but it failed with errors: #{object.errors.full_messages.join(', ')}"
```